cumulusci/core/config/universal_config.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+import yaml
+from pathlib import Path
+
+from cumulusci.core.utils import merge_config
+from cumulusci.core.config.project_config import BaseProjectConfig
+from cumulusci.core.config import BaseTaskFlowConfig
+
+__location__ = os.path.dirname(os.path.realpath(__file__))
+
+
+class UniversalConfig(BaseTaskFlowConfig):
+    """ Base class for the global config which contains all configuration not specific to projects """
+
+    config = None
+    config_filename = "cumulusci.yml"
+    project_config_class = BaseProjectConfig
+
```

Comment:
@prescod btw I got rid of `global_config_dir = ".cumulusci"` here since it was unused.